doc/manual/x509.rst

```diff
@@ -24,10 +24,6 @@ in the :doc:`tls` protocol. A X.509 certificate is represented by
 
      Returns the distinguished name of the issuer
 
- .. cpp:function:: X509_DN subject_dn() const
```

Comment:
This isn't redundant exactly, this second one should be for `issuer_dn` (and then `subject_dn` should be corrected re what name it returns)